CMakeLists.txt

```
@@ -89,6 +89,33 @@ if(APPLE)
 
   # RPATH stuff
   set(CMAKE_MACOSX_RPATH ON)
+  if(NOT IOS)
+    # Determine if we can link against ML Compute
+    set(MLCOMPUTE_FOUND OFF)
+    execute_process(
+      COMMAND bash -c "xcrun --sdk macosx --show-sdk-path"
+      OUTPUT_VARIABLE _macosx_sdk_path
+      OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+    set(_SDK_SEARCH_PATH "${_macosx_sdk_path}/System/Library/Frameworks/")
+    set(_FRAMEWORK_SEARCH_PATH "/System/Library/Frameworks/")
+
+    find_library(_MLCompute_fwrk_path_ NAMES MLCompute REQUIRED PATHS ${_FRAMEWORK_SEARCH_PATH} NO_DEFAULT_PATH)
+    find_library(_MLCompute_sdk_path_ NAMES MLCompute REQUIRED PATHS ${_SDK_SEARCH_PATH} NO_DEFAULT_PATH)
```

Can you please remove `REQUIRED` keyword here, as it will abort execution of the script if framework is not found
```suggestion
    find_library(_MLCompute_fwrk_path_ NAMES MLCompute PATHS ${_FRAMEWORK_SEARCH_PATH} NO_DEFAULT_PATH)
    find_library(_MLCompute_sdk_path_ NAMES MLCompute PATHS ${_SDK_SEARCH_PATH} NO_DEFAULT_PATH)
```

---

done! The check if those were found is a couple lines down. Thank you!